docs/topics/quickstart.md

```diff
@@ -1,10 +1,26 @@
 # Quickstart Guide
 
-This guide goes through all the necessary steps to create a cluster on Hetzner infrastructure (on HCloud & Hetzner Dedicated).
+This guide goes through all the necessary steps to create a cluster on Hetzner infrastructure (on HCloud).
 
-## Preparing Hetzner
+>Note: The cluster templates used in the repository and in this guide for creating clusters are for development purposes only. These templates are not advised to be used in the production environment. However, the software is production-ready and users use it in their production environment. Make your clusters production-ready with the help of Syself Autopilot. For more information, contact <info@syself.com>. 
```

Comment:
I think a hyperlink to the Autopilot landing page would be good here. But afaik this URL does not exist yet.

Reply:
no it does not

Reply:
will update given the page is available now.